ycmd.el

```diff
@@ -254,6 +254,26 @@ use `ycmd-parse-buffer'."
   :group 'ycmd
   :type 'boolean)
 
+(defcustom ycmd-url-proxy-services nil
+  "Specify proxies for local traffic with the ycmd server.
+
+Sets the variable `url-proxy-services' in `ycmd--request' when
+sending requests to the ycmd server:
+
+nil
+    Do not use a proxy server at all.
+
+`default'
+    Use the value of `url-proxy-services'.
+
+Alist
```

Comment:
Do we even need this? I think `nil` or the value of `url-proxy-settings` should be enough.